app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._sqlconnection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(self,
+               first_name: str,
+               last_name: str) -> None:
```

Comment:
move each parameter on a new line including `self`  is a recommended practice for improving code readability,

Reply:
Fix in all places